docs/overview.rst

```diff
@@ -0,0 +1,177 @@
+- **Shell level**: displays the number of nested shells if it starts being too much.
+- **Proxy**: indicate whether a proxy is configured.
+- **Temperature**: warn if the temperature goes too high.
+- **Prompt switch**: command allowing you to rapidely go back to the default prompt, or the even most simple possible.
+- **Wifi**: shows the wireless network signal strength.
+- **Dir stack**: helps you follow the state of this overlooked, but neat, feature of modern shells.
+- **Title**: synchronize the prompt and the terminal's window title (or status bar).
+
+
+Version Control
+---------------
+
+Liquid Prompt has one of the most comprehensive supports for source code management systems living in the shell.
+It has near-to-complete support for:
+
+- **Git**,
+- **Mercurial**,
+- **Subversion**,
+- **Fossil**,
+- **Bazaar**.
+
+It shows the current branch/tags, its state, and several statistics on the current commits/edits.
+
+
+Software Configuration Environments
+-----------------------------------
+
+Modern development environments love to use abstractions on top of software packages.
+Liquid Prompt helps you knowing which one is currently in use where you are:
+
+- **AWS**,
+- **Conda**,
+- **Python venv**,
+- **Kubernetes**,
+- **Terraform**,
+- **Docker**,
+- **Container**,
+- **Node env**,
+- **Ruby venv**,
+- **Software Collections**.
+
+Those show up only if you enter a configured directory, or have configured environment variables.
+
+
+Known Limitations and Bugs
+--------------------------
+
+- Does not display the number of commits to be pushed in Mercurial repositories.
+- Browsing very large Subversion repositories may dramatically slow down
+  the display of Liquid Prompt (use :attr:`LP_DISABLED_VCS_PATHS` to avoid that).
+- Subversion repositories cannot display commits to be pushed because
+  that's not how Subversion works
+- The window's title escape sequence may not work properly on some terminals
+  (like `xterm-256`).
+- The analog clock requires a Unicode-aware terminal and at least a
+  sufficiently complete font on your system. The `Symbola <https://dn-works.com/ufas/>`_
+  font, designed by Georges Douros, is known to work well. On Debian or Ubuntu
+  install try the `fonts-symbola` or `ttf-ancient-fonts` package.
+  "Powerline" and "Nerd" fonts also have the approriate symbols.
+- The "sudo" feature is disabled by default as there is no way to detect
+  if the user has sudo rights without triggering a security alert
+  that will annoy the sysadmin.
+
+
+Competitors
+-----------
+
+All prompt systems tend to focus on some feature sets.
+If you don't like Liquid Prompt's design, you may be interested in one of those popular prompts:
+
+- `Starship <https://starship.rs/>`_: focused on showing development contexts. Probably the largest set of detected languages.
+- `Spaceship <https://spaceship-prompt.sh/>`_: similar spirit to Starship (with a little bit fewer features), but only for Zsh.
```

Comment:
```suggestion
- `Spaceship <https://spaceship-prompt.sh/>`_: similar spirit to Starship (with a few fewer features), but only for Zsh.
```

Reply:
One learn everyday, I would have bet this wasn't correct :laughing: